src/components/GradeItemAccordion.tsx

```diff
@@ -1,9 +1,12 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
+
```

Comment:
Not sure if the empty lines on line 3 and 6 are intentional but just wanted to make sure